### PR TITLE
updated for structure and suggested use cases.

### PR DESCRIPTION
**Pull request by datamarmot**
needs review by eslam and janis

@netcode @jkirsteins Can you help me with the following:
On line 20, add a link or example of the credential stuffing rule that uses threat intel.
On lines 29>31 add links for the sources.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [inline content/en/security/application_security/threats/threat-intelligence.md:30]: If we want to remove this, we can't merge the documentation until it's removed on the backend (I'm starting that process today, but it'll take until end of week)

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [inline content/en/security/application_security/threats/threat-intelligence.md:33]: Do we want to send all of spur to ASM? Cause if not, we should leave the disclaimer here.

Otherwise I foresee customers contacting support that there's a bug, cause a spur feed (with unsupported category) is not surfaced.

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [inline content/en/security/application_security/threats/threat-intelligence.md:32]: These are exact source names as they will appear in the spans, so imho we shouldn't change the capitalization.

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (datamarmot) [reply content/en/security/application_security/threats/threat-intelligence.md:30]: I think its okay to have it as an undocumented source for a few days.. It is low traffic and it was undocumented previously. I defer to your comfort level on that.

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (datamarmot) [reply content/en/security/application_security/threats/threat-intelligence.md:33]: Makes sense. Need to update it to malware and residential proxies.

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [reply content/en/security/application_security/threats/threat-intelligence.md:30]: Compromise: ok with me to merge this once we've gotten the approval for the change from event platform (but before it is deployed, which takes a day and happens on Tuesdays/Thursdays)

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [reply content/en/security/application_security/threats/threat-intelligence.md:33]: https://github.com/DataDog/logs-backend/pull/59572 Removing minerstat, and adding residential_proxy

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [reply content/en/security/application_security/threats/threat-intelligence.md:32]:
```suggestion
- Tor Exit Nodes
```

Comment (datamarmot) [reply content/en/security/application_security/threats/threat-intelligence.md:32]: Spur is the proper noun representation which I think we should reflect.
Exit nodes is more grammatically correct. Its not a hill I'm willing to die on. Defer to you, but that is the reasoning for changing.

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (netcode) [inline content/en/security/application_security/threats/threat-intelligence.md:20]:
```suggestion
1. Reducing detection rule thresholds for business logic threats such as credential stuffing. For example Users can clone [credential stuffing default rule](https://app.datadoghq.com/security/configuration/asm/rules/view/wnp-zlu-woa) and modify it to meet their needs.
```

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (netcode) [inline content/en/security/application_security/threats/threat-intelligence.md:29]:
```suggestion
- [abuse.ch](https://threatfox-api.abuse.ch)
```

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (netcode) [inline content/en/security/application_security/threats/threat-intelligence.md:30]:
```suggestion
- [FireHOL](https://iplists.firehol.org/)
```

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [reply content/en/security/application_security/threats/threat-intelligence.md:32]: The search is case sensitive, which is why I'm pushing back on this. If users copy-paste the capitalized versions, they will not be able to use it in a query.

Compare these two searches (same time interval, only difference is the capitalization):
- [lowercase spur](https://app.datadoghq.com/security/appsec/traces?query=%40threat_intel.results.source.name%3Aspur%20&start=1698474264130&end=1698525206485&paused=true)
- [capitalized spur](https://app.datadoghq.com/security/appsec/traces?query=%40threat_intel.results.source.name%3ASpur%20&start=1698474264130&end=1698525206485&paused=true)

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (jkirsteins) [reply content/en/security/application_security/threats/threat-intelligence.md:33]: @datamarmot We have to add `residential_proxy` separately. I tested in staging over the last 2 days, and it will add a lot of traffic (5x the existing feeds combined). I'd like to add some way to roll that out progressively, and i'll update the docs subsequently again.

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (aliciascott) [inline content/en/security/application_security/threats/threat-intelligence.md:15]:
```suggestion
Datadog provides built-in [threat intelligence][1] datasets for Application Security Management (ASM) and Cloud SIEM (Security Information and Event Management). This provides additional insights to take action when security activity is observed.
```

File: content/en/security/application_security/threats/threat-intelligence.md
Comment (aliciascott) [inline content/en/security/application_security/threats/threat-intelligence.md:23]:
```suggestion
Datadog recommends _against_ the following:
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Just a couple minor things otherwise looks good!

**Pull request merged**
